elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -296,8 +299,10 @@ public boolean matchesId(String checkId) {
         }
 
         Iterable<T> list;
-        FilterScope filterScope = new FilterScope(requestScope, loadClass);
-        list = tx.loadObjects(loadClass, filterScope);
+        Optional<FilterExpression> filterExpression = requestScope.getLoadFilterExpression(loadClass);
+
+        list = (Iterable<T>) tx.loadObjects(loadClass, filterExpression,
```

Comment:
Don't we need to get the FilterExpression for the permissions and join them with the request filter expressions?